modules/serverless-logic-install-kn-workflow-plugin-cli.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/install-serverless-operator.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-plugin-cli_{context}"]
+= Installing the {ServerlessLogicOperatorName} using the Knative workflow plugin from the CLI
+
+OpenShift Serverless Logic Workflow provides a plugin named `kn-plugin-workflow` for the Knative CLI, enabling you to set up a local workflow project using the command line.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+
+.Procedure
+
+. Download the latest binary file.
+
+. Install the `kn workflow` command as a plug-in of the Knative CLI using the following steps:
+
+.. Copy the `kn-workflow` binary file to a directory in your PATH, such as `/usr/local/bin`, and ensure the file name is `kn-workflow`:
++
+[source,terminal]
+----
+cp path/to/downloaded/kn-workflow /usr/local/bin/kn-workflow
```

Comment:
We include the prompt in the command (is "prompt" the right word?).
```suggestion
$ cp path/to/downloaded/kn-workflow /usr/local/bin/kn-workflow
```

Reply:
Yes, I have added $